myst_nbs/diagnostics_and_criticism/model_averaging.myst.md

```diff
@@ -47,17 +56,16 @@ az.style.use("arviz-darkgrid")
 
 +++ {"papermill": {"duration": 0.068882, "end_time": "2020-11-29T12:13:08.020372", "exception": false, "start_time": "2020-11-29T12:13:07.951490", "status": "completed"}, "tags": []}
 
-# Model averaging
-
-When confronted with more than one model we have several options. One of them is to perform model selection, using for example a given Information Criterion as exemplified [in this notebook](model_comparison.ipynb) and this other [example](GLM-model-selection.ipynb). Model selection is appealing for its simplicity, but we are discarding information about the uncertainty in our models. This is somehow similar to computing the full posterior and then just keep a point-estimate like the posterior mean; we may become overconfident of what we really know.
+When confronted with more than one model we have several options. One of them is to perform model selection, using for example a given Information Criterion as exemplified the [model comparison notebook](https://www.pymc.io/projects/docs/en/stable/learn/core_notebooks/model_comparison.html) and the [GLM model selection notebook](https://www.pymc.io/projects/examples/en/latest/generalized_linear_models/GLM-model-selection.html). Model selection is appealing for its simplicity, but we are discarding information about the uncertainty in our models. This is somehow similar to computing the full posterior and then just keep a point-estimate like the posterior mean; we may become overconfident of what we really know.
```

Comment:
This should be breaking CI but it's not :/, we should update the pre-commit config, there are several domains missing from the list of forbidden domains:

https://github.com/pymc-devs/pymc-examples/blob/76fae3ffff138021abf484ea4317e74d31f970cb/.pre-commit-config.yaml#L87-L97

can you add `www.pymc.io` and `docs.scipy.org/doc` to the list?

All these are domains of documentation for python packages built with sphinx. Therefore, we should not use urls to link there but sphinx cross-references, there are multiple examples on how to use references in the [material for my webinar](https://pymc-data-umbrella.xyz/en/latest/about/contributing_to_documentation/docs_presentation.html#cross-referencing-roles). The style makes sure all notebooks have the `(notebook_id)=` part right at the top so we can link between notebooks using the `ref` role. 

I searched for the correct urls and targets in https://github.com/pymc-devs/pymc-examples/pull/414#issuecomment-1210092569, the urls there should not be used, only the targets. I only added the urls for completeness so you could check the preview was indeed generating the correct link.